extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -71,13 +70,12 @@ export class ProjectsController {
 					text = constants.InProgress;
 				}
 
-				let infoRow: (string | dataworkspace.IconCellValue)[] = [count.toString(),
+				let infoRow: (string | dataworkspace.IconCellValue)[] = [infoRows.length.toString(),
```

Comment:
should the id be something stored with the deploy info? The id isn't supposed change, so it can be set when the deployInfo is first created. It would also be good to store the id for when this dashboard data gets persisted later. Same for build info

Reply:
As discussed in the meeting, removed the ID column.